src/fty_security_wallet_server.cc

```diff
@@ -28,17 +28,24 @@
 
 #include "fty_security_wallet_classes.h"
 
-#include "secw_exception.h"
-#include "secw_security_wallet.h"
-#include "secw_helpers.h"
+//Too old glibc :(. <gettid> is available since glibc 2.30
```

Comment:
there should be an AC_CHECK_FUNCS() call to see if getid() is available or not, so that this code block is conditionally defined. This will otherwise hit us when switching to more recent glibc. (hint: use acinclude.m4...)

Reply:
Resolved by basic #if guard